lib/typescript/httpclient/model/Template.ts

```diff
@@ -0,0 +1,9 @@
+import {MessageSource} from './Message';
+
+export interface Template {
+  id: string;
+  name: string;
+  sourceType: MessageSource;
```

Comment:
The naming here is off (not sure it was already like that... but we also noticed when discussing channels yesterday). I suppose this should be called `SourceType`

Reply:
It's too late since the backend PR was already merged, but I just reckoned that since this field is called `source` everywhere else this should also be the case for the field on the templates.